deeppavlov/requirements/kbqa.txt

```diff
@@ -0,0 +1,5 @@
+rapidfuzz==0.3.0
```

Comment:
rapidfuzz is referenced in two `.txt` files. Maybe remove this line and depend also on `rapidfuzz.txt` in configs where `kbqa.txt` is used?

Reply:
kbqa.txt is not used and I deleted it.